src/modal/classes.scss

```diff
@@ -5,8 +5,24 @@
 .iui-modal {
   @include iui-modal;
 
-  &-full-page {
-    @include iui-modal-full-page;
+  &-backdrop {
```

Comment:
Why did you remove `&-full-page`?

Reply:
The previous mixin was rolled into the mixin for `.iui-modal` along with resizable.

Reply:
So we actually just worked on making `iui-modal-default`, `iui-modal-full-page`, and `iui-modal-resizable` into their own mixins also. We did this so that resizable modals won't need to have both of the classnames `iui-modal-default` and `iui-modal-resizable`, as `iui-modal-default` is now included into `iui-modal-resizable` in classes.scss.